lexicons/place/stream/default/metadata.json

```diff
@@ -0,0 +1,166 @@
+      "record": {
+        "type": "object",
+        "properties": {
+          "contentWarnings": {
+            "type": "array",
+            "items": {
+              "type": "string",
+              "knownValues": [
+                "place.stream.default.metadata#death",
+                "place.stream.default.metadata#drugUse",
+                "place.stream.default.metadata#fantasyViolence",
+                "place.stream.default.metadata#flashingLights",
+                "place.stream.default.metadata#language",
+                "place.stream.default.metadata#nudity",
+                "place.stream.default.metadata#PII",
+                "place.stream.default.metadata#sexuality",
+                "place.stream.default.metadata#suffering",
+                "place.stream.default.metadata#violence"
+              ]
+            },
+            "description": "Content warnings for this stream."
+          },
+          "contentRights": {
+            "type": "ref",
+            "ref": "#contentRights"
+          },
+          "distributionPolicy": {
+            "type": "ref",
+            "ref": "#distributionPolicy"
+          }
+        }
+      }
+    },
+    "contentRights": {
+      "type": "object",
+      "description": "Content rights and attribution information.",
+      "properties": {
+        "creator": {
+          "type": "string",
+          "description": "Name of the creator of the work."
+        },
+        "copyrightNotice": {
+          "type": "string",
+          "description": "Copyright notice for the work."
+        },
+        "copyrightYear": {
+          "type": "integer",
+          "description": "Year of creation or publication."
+        },
+        "license": {
+          "type": "string",
+          "description": "License URL or identifier.",
+          "knownValues": [
+            "place.stream.default.metadata#all-rights-reserved",
+            "place.stream.default.metadata#cc0_1__0",
+            "place.stream.default.metadata#cc-by_4_00",
+            "place.stream.default.metadata#cc-by-sa_4__0",
+            "place.stream.default.metadata#cc-by-nc_4__0",
+            "place.stream.default.metadata#cc-by-nc-sa_4__0",
+            "place.stream.default.metadata#cc-by-nd_4__0",
+            "place.stream.default.metadata#cc-by-nc-nd_4__0"
+          ]
+        },
+        "creditLine": {
+          "type": "string",
+          "description": "Credit line for the work."
+        }
+      }
+    },
+    "distributionPolicy": {
+      "type": "object",
+      "description": "Distribution and rebroadcast policy.",
+      "properties": {
+        "allowArchive": {
+          "type": "boolean",
+          "description": "Whether nodes can archive this stream."
+        },
+        "broadcastExpiry": {
+          "type": "string",
+          "format": "datetime",
+          "description": "When rebroadcast permissions expire. If not specified, there is no expiration on rebroadcast permission."
+        }
```

Comment:
I think these are one and the same - if you don't want archival, set a five-minute expiry.

Also leaning toward `deleteAfter` to make the syntax here really clear